### ISSUE TITLE
addField putting a i on the end of int fields which causes a write error

### ISSUE THREAD

**Issue by kb57442** [offa/influxdb-cxx#133]
Using influx 1.8.10 and the most recent version of influxdb-cxx
Writing this statement fails
` db->write(influxdb::Point{"MyValues"}.addTag("label", "test").addField("pointValue",1 ));`
But this one succeeds: 
`db->write(influxdb::Point{"MyValues"}.addTag("label",` "test").addField("pointValue",1.00 ));`

When doing a tcpdump on the server it shows that the first statement trys to insert using a pointValue=1i

The second statement just shows pointValue=1.00

The error returned by influxdb is "invalid field format"

**Comment by offa** [comment]
Thanks for reporting. This is odd, especially since there's a [system test which tests almost the same values](https://github.com/offa/influxdb-cxx/blob/master/test/system/InfluxDBST.cxx#L125).

Appending an `i` to integer types is [correct behaviour](https://docs.influxdata.com/influxdb/v1.8/write_protocols/line_protocol_tutorial/#data-types).

**Comment by offa** [comment]
Could you provide a minimal example to reproduce the error?

**Comment by offa** [comment]
Is this still an issue?

**Comment by kb57442** [comment]
No, The issue was due to a field name in one testing run being specified as a tag in a different testing run.  When one was inserted when it had already been specified as the other it generated the error.  Sorry for the false alarm. 
    On Friday, October 7, 2022 at 07:12:53 AM PDT, offa ***@***.***> wrote:  
 
 


Is this still an issue?
